examples/siro_sandbox/hb_config_helper.py

```diff
@@ -40,52 +39,26 @@ def update_config_and_args(
             )
             agent_key = "" if len(sim_config.agents) == 1 else "agent_0_"
             agent_sensor_name = f"{agent_key}third_rgb"
-            args.debug_images.append(agent_sensor_name)
+            hitl_config.debug_images.append(agent_sensor_name)
             gym_obs_keys.append(agent_sensor_name)
 
-        # Code below is ported from interactive_play.py. I'm not sure what it is for.
-        if True:
```

Comment:
I investigated and this `if True: ...` block is mostly not needed. I also think it's messy to override the config at runtime. In the case of `must_call_stop`, it turns out that the `rearrange` app needs this, so `hitl_rearrange.yaml` includes this config override.